src/main/java/pl/javastart/task/ComponentParameters.java

```diff
@@ -0,0 +1,35 @@
+package pl.javastart.task;
+
+public class ComponentParameters {
+
+    static final int ADDITIONAL_FREQUENCY = 100;
+    private int frequency;
+    private int optimalTemp;
```

Comment:
Myślę, że lepszą nazwą byłoby "currentTemp", albo po prostu temperature, bo to aktualna temperatura, a nie optymalna temperatura